experimental/table/server/master/src/main/java/alluxio/master/catalog/CatalogMaster.java

```diff
@@ -101,4 +103,17 @@ boolean attachDatabase(String dbName, String dbType, CatalogConfiguration config
    */
   List<Partition> readTable(String dbName, String tableName, Constraint constraint)
       throws IOException;
+
+  /**
+   * Get statistics on the partitions.
+   *
+   * @param dbName database name
+   * @param tableName table name
+   * @param partNamesList partition names
+   * @param colNamesList column names
+   * @return a map mapping partitions to a list of column statistics info
```

Comment:
the key is a string - is it the partition name?

Reply:
yes, updated javadoc